### PR TITLE
Fix concurrent to 0.9.1

### PR DESCRIPTION
**Pull request by ph**

### CODE REVIEW

**Comment by untergeek** [comment]
LGTM™

**Comment by suyograo** [comment]
LGTM

File: logstash-input-beats.gemspec [hunk 1]
```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency "logstash-core", ">= 1.5.4", "< 3.0.0"
 
   s.add_runtime_dependency "logstash-codec-plain"
-  s.add_runtime_dependency "concurrent-ruby"
+  s.add_runtime_dependency "concurrent-ruby", "0.9.1"
```
Comment (urso) [inline logstash-input-beats.gemspec]: phew, all these dependencies make my head spin.

**Comment by urso** [comment]
we need to bump plugin version?

**Comment by suyograo** [comment]
yep @ph is doing it in another PR

**Comment by ph** [comment]
@urso just pushed 0.9.5 with the version pin to 0.9.1

**Pull request closed without merging**